warehouse/packaging/forms.py

```diff
@@ -0,0 +1,41 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import wtforms
+
+from warehouse import forms
+from warehouse.i18n import localize as _
+
+
+class SubmitMalwareObservationForm(forms.Form):
+    """Form to submit details about a Project with Malware"""
+
+    # TODO: Add a honeypot field? Or better to add a CAPTCHA on the page?
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** I didn't want to import the HoneypotMixin from `accounts/forms.py` since that felt like it was traversing a boundary. I can definitely refactor that to a common spot.
‏
I guess since folks must be authenticated already, we don't need another verification form here?

Reply:
Agreed. Will remove comment, and we can add mitigations later if needed.